src/Options.php

```diff
@@ -662,7 +662,7 @@ private function configureOptions(OptionsResolver $resolver): void
         $resolver->setAllowedValues('dsn', \Closure::fromCallable([$this, 'validateDsnOption']));
         $resolver->setAllowedValues('integrations', \Closure::fromCallable([$this, 'validateIntegrationsOption']));
         $resolver->setAllowedValues('max_breadcrumbs', function ($value) {
```

Comment:
Please add a typehint to this function for the return type and also extract its code to a `private function` like I started doing for the other validation callables